<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAnime3Class</class>
 <widget class="QMainWindow" name="QAnime3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1520</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1520</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qAnime3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="vLayoutButtons">
      <item>
       <widget class="QPushButton" name="buttonScan">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vLayoutTable">
      <item>
       <widget class="QTreeWidget" name="treeTorrents">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConfirmRename">
        <property name="text">
         <string>Confirm rename</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
